graphql/string.go

```diff
@@ -6,9 +6,41 @@ import (
 	"strconv"
 )
 
+const alphabet = "0123456789ABCDEF"
```

Comment:
just a small thing but maybe call this `unicodeAlphabet` or something similar as I could see someone looking at the var name and wondering why it doesn't encompass the whole alphabet.